rir/src/ir/BC_inc.h

```diff
@@ -285,9 +285,10 @@ class BC {
     inline static BC isfun();
     inline static BC invisible();
     inline static BC visible();
-    inline static BC extract1();
-    inline static BC subset1();
-    inline static BC extract2();
+    inline static BC extract1_1();
+    inline static BC extract1_2();
+    inline static BC extract2_1();
+    inline static BC extract2_2();
     inline static BC subset2();
```

Comment:
Is the `subset2` just a leftover from before?

Reply:
oh, yeah, that should have been deleted too, thanks